.editorconfig

```diff
@@ -0,0 +1,2 @@
+indent_style=tab
+indent_size=4
```

Comment:
My editor was complaining about indents not being spaces.  This defines it formally.